packages/aws-cdk-lib/aws-apigateway/test/lambda-api.test.ts

```diff
@@ -299,7 +299,7 @@ describe('lambda api', () => {
       ResourceId: { Ref: 'lambdarestapiproxyE3AE07E3' },
       AuthorizationType: 'NONE',
       AuthorizerId: Match.absent(),
-      ApiKeyRequired: Match.absent(),
+      ApiKeyRequired: false,
```

Comment:
This should be the correct value given that it's set [here](https://github.com/aws/aws-cdk/blob/5673b75c7ac1d4e122fd441c43cf42703047c6e0/packages/aws-cdk-lib/aws-apigateway/lib/resource.ts#L303)